com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -681,7 +684,6 @@ internal void OnDespawnObject(NetworkObject networkObject, bool destroyGameObjec
                 if (NetworkManager.PrefabHandler.ContainsHandler(networkObject))
                 {
                     NetworkManager.PrefabHandler.HandleNetworkPrefabDestroy(networkObject);
-                    OnDespawnObject(networkObject, false);
```

Comment:
@LukeStampfli , I could you double check this?  I did some testing on my end and I think this is no longer needed with the more recent updates (I think this was for some message ordering issue, but that should be fixed now).
Gladly will revert this is you think there is a special case for having that OnDespawnObject here.

Reply:
I tried understand this but not sure why we had it. It does a recursive call into OnDespawnObject again for some reason I can't find a reason why we would want to do that.